src/modality-specific-files/genetic-descriptor.md

```diff
@@ -19,11 +19,12 @@ and can be used for practical guidance when curating a new dataset.
 
 ## Dataset Description
 
-Genetic descriptors are encoded as an additional, OPTIONAL entry in the
+If genetic data is supplied as part of a BIDS dataset,
+genetic descriptors are encoded as an additional, REQUIRED entry in the
```

Comment:
Because there are no 'data' everything is optional -- but I guess the point here is you must use this and this en encode the genetic information available ; if that's the case then fine

Reply:
```suggestion
these "genetic descriptors" are encoded as an additional, REQUIRED entry in the
```